src/app.ts

```diff
@@ -76,7 +89,8 @@ async function app() {
 
   let resources: Package;
   try {
-    resources = await getResources(inDir, defs);
+    resources = await getResources(processor);
+    resources.add(config);
```

Comment:
So... when I ran this against US Core I got some unexpected results.  After some investigation, I realized it is because the processor may need the config when it's processing.  In the `master` branch, the config is added to the `Package` (represented by the `resources` variable) _before_ it does any processing.  In this PR branch, it is added _after_ it does the processing (so the processing does not benefit from it).

You'll either need to pass the `config` around in a few function calls in order to get it into the `Package` prior to processing, or you need to construct the package here and pass _it_ around instead.  Either way, we're going to need to pass some stuff around.

If you're trying this yourself, here's what tipped me off to the problem.  If you run the `master` branch against US Core and then this branch against US Core, there will only be a few differences in the output -- but the differences will be that this branch does not remove the extension URL assignment rules that _should_ be removed by the `RemoveExtensionURLAssignmentRules` optimizer.

<img width="1578" alt="image" src="https://user-images.githubusercontent.com/2278253/100805020-e6633000-33fb-11eb-9814-8ba8fcee33a1.png">